transcom/mymove
# B-20616 Add mobile shipment customer flows int

## [B-20616](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A982502)

## Summary

This PR adds the ability to create, edit and soft delete mobile home shipments.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Login as a customer
2. Create a move
3. Add a mobile home shipment
4. Confirm the shipment has been added to the list with the right styling
5. Confirm ability to delete and edit your added mobile home shipment(s).

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
_**Mobile Home added to shipment options**_
<img width="971" alt="Screenshot 2024-08-26 at 5 50 06 AM" src="https://github.com/user-attachments/assets/754667d5-a384-47b4-a8dc-68002cc1fa1f">

_**Mobile Home Info Page**_
<img width="829" alt="Screenshot 2024-08-26 at 5 50 13 AM" src="https://github.com/user-attachments/assets/fe205736-9046-45a6-a70c-3dd5af80e65a">

_**Incomplete Mobile Home added to shipment list**_
<img width="980" alt="Screenshot 2024-08-27 at 3 32 48 PM" src="https://github.com/user-attachments/assets/7f1fe10f-1de7-4fbf-92b9-46c730bbeb99">

_**ANDI SCREENSHOTS**_
<img width="1705" alt="Screenshot 2024-08-27 at 2 19 23 PM" src="https://github.com/user-attachments/assets/5d8db8c8-5b83-4626-b3d3-5ad594985eca">
<img width="1440" alt="Screenshot 2024-08-27 at 2 18 04 PM" src="https://github.com/user-attachments/assets/ad7977c7-a3c3-4926-9a4e-435dd93419a3">
<img width="1427" alt="Screenshot 2024-08-27 at 2 16 55 PM" src="https://github.com/user-attachments/assets/825d2a50-2bba-4320-a746-1cf7059c12ad">
<img width="1434" alt="Screenshot 2024-08-27 at 2 10 08 PM" src="https://github.com/user-attachments/assets/9202c446-90c6-4309-9faf-28a826a71b0f">
